lib/tasks/scrape.rake.copy.rb

```diff
@@ -0,0 +1,159 @@
+namespace :scrape do 
```

Comment:
It's better to keep your files clean. Lets store it somewhere outside the project and delete it here.